### PR TITLE
Add import indent

### PR DESCRIPTION
**Pull request by femtotrader**
Try to fix https://github.com/manahl/arctic/issues/49

### CODE REVIEW

**Comment by jamesblackburn** [comment]
Thanks! Will need a unit test too which exercises the broken code

**Comment by femtotrader** [comment]
It should be ok... Please tell me is something is missing

**Comment by bmoscon** [comment]
You need to add a unit test for the broken code. The test should fail without your fix and pass with it.

**Comment by femtotrader** [comment]
This is the case

see test_stringify_tickstore_library

File: tests/integration/test_arctic.py [hunk 1]
```diff
@@ -144,3 +144,9 @@ def test_default_mongo_retry_timout():
     with pytest.raises(LibraryNotFoundException):
         Arctic('unresolved-host', serverSelectionTimeoutMS=0)['some.lib']
     assert time.time() - now < 1.
+
+def test_stringify_tickstore_library(mongo_host):
+    # Fix GH issue 49 - str(tick library) fails in IPython
+    store = Arctic(mongo_host=mongo_host)
+    store.initialize_library('library_name', 'TickStoreV3')
+    s = str(store['library_name'])
```
Comment (AdrianTeng) [inline tests/integration/test_arctic.py]: `assert 'arctic.library_name' in str(store['library_name'])` would be nicer

**Comment by AdrianTeng** [comment]
LGTM

**Comment by femtotrader** [comment]
assert 'arctic.library_name' in str(store['library_name'])

added and commit squashed

File: tests/integration/test_arctic.py [hunk 2]
```diff
@@ -144,3 +144,9 @@ def test_default_mongo_retry_timout():
     with pytest.raises(LibraryNotFoundException):
         Arctic('unresolved-host', serverSelectionTimeoutMS=0)['some.lib']
     assert time.time() - now < 1.
+
+def test_stringify_tickstore_library(mongo_host):
```
Comment (jamesblackburn) [inline tests/integration/test_arctic.py]: This test should really live on a test file with the sae name as the module under test.  `test_tickstore.py` or similar...

Comment (femtotrader) [inline tests/integration/test_arctic.py]: sorry... @AdrianTeng merged it ... do you want I move this test or will you do it ?

Comment (AdrianTeng) [inline tests/integration/test_arctic.py]: I'll do it

Comment (jamesblackburn) [inline tests/integration/test_arctic.py]: Could you make it a unit test, too :)

**Pull request merged**